crux-llvm/test-data/golden/funptr-array.good

```diff
@@ -0,0 +1,36 @@
+[Crux] Counter example for /home/kquick/work/crux-sugar/crux-llvm/test-data/golden/funptr-array.c:12:12: error: in mytestfunction
```

Comment:
Is this supposed to be part of the file, given it has paths from your env in it?